debian/pop-desktop.gsettings-override

```diff
@@ -19,8 +19,10 @@ session-name = "pop"
 logo = ''
 
 [org.gnome.settings-daemon.plugins.power]
```

Comment:
Probably worth keeping the Title 20 note.

Reply:
Added the title 20 note, and added the suspend settings to raspberry pi.